core/src/main/java/com/graphhopper/routing/util/EncodingManager.java

```diff
@@ -349,7 +349,10 @@ public static boolean isValidEncodedValue(String name) {
             if (c == '_') {
                 if (underscoreCount > 0) return false;
                 underscoreCount++;
-            } else if (!isLowerLetter(c) && !isNumber(c)) {
+            } else if (c == '-'){
```

Comment:
Very nice catch, but won't that exclude all tags which have more than one hyphen?

Reply:
This loops through every character inside the name. I did test it on my side for all 8 Heatmap tags. Where 7 of the 8 contain more than 1 dash. It is working on my end.

Reply:
alright :+1: